src/js/queries/newsPageContentFragment.js

```diff
@@ -0,0 +1,20 @@
+const newsPageContentFragment = `
```

Comment:
What does newsPageContentFragment get used for?

Reply:
newsPageFragment was originally on JanisBasePageNode https://github.com/cityofaustin/janis/pull/766/commits/efdbec002a332583fbace5363df1e30651ae7b8b, this was to have one on NewsPageNode. Since newsPageFragment is now on NewsPageNode, I will delete this one